manifests/buckets.yaml

```diff
@@ -3,7 +3,7 @@ kind: ObjectBucketClaim
 metadata:
   name: black-flake
```

Comment:
you'd need to temporarily change this name (or disable the ArgoCD autosync, delete the claim manually, wait for the operator to reconcile and then apply this as a new claim)

The bucket name can't be changed on the fly.

Reply:
Yes, I know about that problem. Thx.